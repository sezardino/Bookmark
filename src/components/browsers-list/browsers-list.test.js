import React from 'react';
import {shallow} from 'enzyme';

import BrowsersList from './browsers-list';
describe('Component BrowsersList', () => {
	let mockData = [
		{id: 1, name: 'Some', minVersion: 11},
		{id: 2, name: 'Some', minVersion: 22},
		{id: 3, name: 'Some', minVersion: 33},
	];
	const component = shallow(<BrowsersList data={mockData} />);

	it('Component BrowsersList rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
