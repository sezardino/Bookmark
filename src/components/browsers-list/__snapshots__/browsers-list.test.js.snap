// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component BrowsersList Component BrowsersList rendered correctly 1`] = `
"<ul className=\\"download__cards cards\\">
  <li className=\\"cards__item\\">
    <p className=\\"img-wrapper cards__img\\">
      <img src={[undefined]} alt=\\"Some logotype\\" />
    </p>
    <h3 className=\\"cards__title title\\">
      Add to Chrome
    </h3>
    <p className=\\"cards__subtitle\\">
      Minimum version 
      11
    </p>
    <Link to=\\"/extension\\" className=\\"button button--blue button--cards cards__button\\">
      Add &amp; Instal Extension
    </Link>
  </li>
  <li className=\\"cards__item\\">
    <p className=\\"img-wrapper cards__img\\">
      <img src={[undefined]} alt=\\"Some logotype\\" />
    </p>
    <h3 className=\\"cards__title title\\">
      Add to Chrome
    </h3>
    <p className=\\"cards__subtitle\\">
      Minimum version 
      22
    </p>
    <Link to=\\"/extension\\" className=\\"button button--blue button--cards cards__button\\">
      Add &amp; Instal Extension
    </Link>
  </li>
  <li className=\\"cards__item\\">
    <p className=\\"img-wrapper cards__img\\">
      <img src={[undefined]} alt=\\"Some logotype\\" />
    </p>
    <h3 className=\\"cards__title title\\">
      Add to Chrome
    </h3>
    <p className=\\"cards__subtitle\\">
      Minimum version 
      33
    </p>
    <Link to=\\"/extension\\" className=\\"button button--blue button--cards cards__button\\">
      Add &amp; Instal Extension
    </Link>
  </li>
</ul>"
`;
